plugins/woocommerce/includes/class-wc-frontend-scripts.php

```diff
@@ -45,10 +45,31 @@ class WC_Frontend_Scripts {
 	 */
 	public static function init() {
 		add_action( 'wp_enqueue_scripts', array( __CLASS__, 'load_scripts' ) );
+		add_action( 'wp_enqueue_scripts', array( __CLASS__, 'warn_deprecated_scripts' ) );
 		add_action( 'wp_print_scripts', array( __CLASS__, 'localize_printed_scripts' ), 5 );
 		add_action( 'wp_print_footer_scripts', array( __CLASS__, 'localize_printed_scripts' ), 5 );
 	}
 
+	/**
+	 * Log a warning for deprecated scripts/styles that will be removed.
+	 */
+	public static function warn_deprecated_scripts() {
```

Comment:
Should we also add this for admin scripts (and not just the frontend)?

Reply:
Yep good point will add.

Reply:
https://github.com/woocommerce/woocommerce/pull/36576/commits/586a5d8851b35de9a6886a76e1d658d65b8ebb25